### DISCUSSION TITLE
What would the custom command be to launch the start menu?

### DISCUSSION THREAD

**Discussion by stephanlamoureux** [vhanla/winxcorners#41] [General]
I would like to make the bottom left corner launch the Windows start menu.

**Comment by Eichen97** [comment]
Make a start.vbs file with this content:
```
set wShell=wscript.createobject("wscript.shell") 
wShell.sendkeys "^{ESC}"
Set WshShell = Nothing
```
if you double click it, it will pop up the start menu, so you just have to put the route to the file as your custom command path.

**Comment by vhanla** [comment]
New version (which I will launch soon) already includes Start Menu option, as well than other improvements, now for Windows 11 too.

Meanwhile you can use https://github.com/vhanla/StartScreenLauncher compile with http://flatassembler.net/download.php 
but here it is a compiled one, 4KB, mostly the icon file increments its size
[Win8StartScreenLauncher.zip](https://github.com/vhanla/winxcorners/files/11637517/Win8StartScreenLauncher.zip)
just decompress it into any location and add its full executable path as custom command.

**Reply by Eichen97** [reply]
Thanks! your method looks more concistent and reliable.
Would you add the new notification area + calendar as an option? 
Given that the action center is great, but each time i miss clicking a notification, it goes into the void, since i have the taskbar 100% hidden with taskbar magic; and the only fix i got since was using explorer patcher to bind it to WIN+C

**Reply by stephanlamoureux** [reply]
Awesome! Excited about the new version 👍

**Reply by vhanla** [reply]
> Thanks! your method looks more concistent and reliable. Would you add the new notification area + calendar as an option? Given that the action center is great, but each time i miss clicking a notification, it goes into the void, since i have the taskbar 100% hidden with taskbar magic; and the only fix i got since was using explorer patcher to bind it to WIN+C

Win+C on Windows 11 is Copilot now. 🤔

**Comment by stephanlamoureux** [comment]
> New version (which I will launch soon) already includes Start Menu option, as well than other improvements, now for Windows 11 too.
> 
> Meanwhile you can use https://github.com/vhanla/StartScreenLauncher compile with http://flatassembler.net/download.php but here it is a compiled one, 4KB, mostly the icon file increments its size [Win8StartScreenLauncher.zip](https://github.com/vhanla/winxcorners/files/11637517/Win8StartScreenLauncher.zip) just decompress it into any location and add its full executable path as custom command.

Is there still a new version coming out?
